ghu_web/ghu_main/static/ghu_main/css/base.css

```diff
@@ -54,3 +58,32 @@
     background: #4aaec6; /* Fallback */
     background: linear-gradient(#4aaec6, #2dc5ea);
 }
+
+/* Mobile Size */
+@media (max-width: 770px ) {
+    #logo {
+        margin-top: 18px;
+        margin-bottom: 18px;
+    }
+
+    #logo a {
+        font-size: 5.2vw;
```

Comment:
I'm not sure how I feel about the `vw` units in this PR. For narrow viewports, they can make text unreadably small:

![screen shot 2017-09-18 at 22 41 42](https://user-images.githubusercontent.com/431756/30573643-0cf846a2-9cc3-11e7-989a-b014bd4a0a3b.png)

Decreasing the font size here is a god idea, I think, but I think we should set it to some `px`, not make it be a percentage of the viewport width.

Reply:
Yea I agree. I implement codes base on iphone 6. that's why I wonder if we need to add codes for every phones. Ill fix it. Thanks!